### ISSUE TITLE
Add a few examples for generating navigation HTML

### ISSUE THREAD

**Issue by matthiask** [feincms/feincms#566]
... idiomatic, without too many DB queries.

**Comment by joshuajonah** [comment]
Isn't this more django-mptt territory? The `{% recursetree %}` tag is mind-bogglingly novel.

**Comment by matthiask** [comment]
Yes, it is django-mptt territory. We still might want to add a few examples to our documentation as well.
